### ISSUE TITLE
Netty Error when approaching portal

### ISSUE THREAD

**Issue by hellot1m** [Johni0702/BetterPortals#136]
Have not tested this on single player, have only used this mod on our server. Everything is fine until we get near a nether portal. the console goes crazy with netty errors causing server lag. i have turned on advanced reporting and below is a snip of the error.

Full Error: https://pastebin.com/CsDbk23y

BTW this is what i get on the client side around the same time as the netty error on the server:

```
[20:46:01] [Client thread/WARN]: Shader betterportals:render_portalcould not find sampler named depthSampler in the specified shader program.
[20:46:01] [Client thread/INFO]: [biomesoplenty.common.world.BiomeProviderBOPHell:<init>:32]: settings for hell world: 
[20:46:01] [Client thread/INFO]: Creating Nether SeasonInfo for dimension the_nether
[20:46:01] [Client thread/INFO]: Setting weather renderer for dimension [the_nether]
[20:46:02] [Client thread/WARN]: Dropping packet net.minecraft.network.play.client.CPacketConfirmTeleport@6676e9c4
[20:46:02] [Client thread/WARN]: Dropping packet net.minecraft.network.play.client.CPacketPlayer$PositionRotation@be8fd2a
[20:53:16] [Client thread/INFO]: [biomesoplenty.common.world.BiomeProviderBOPHell:<init>:32]: settings for hell world: 
[20:53:16] [Client thread/INFO]: Creating Nether SeasonInfo for dimension the_nether
[20:53:16] [Client thread/INFO]: Setting weather renderer for dimension [the_nether]```

**Comment by Johni0702** [comment]
I'm unable to reproduce the issue myself (tbh I wouldn't have been surprised if it was just BP on a dedicated server, it's been probably over a month since I last tried that). Instead it's probably related to one of your mods. Could you please
1. check if it happens in SP (debugging SP is way easier for me)
2. remove as many third-party mods as possible until you're left with only one (or none)
3. then post the full log

**Comment by hellot1m** [comment]
I will try this. I'm trying to get a few friends who are way better at debugging than I am. In the meantime, below is the modpack that we made along with updates configs. Maybe there is a mod you see that could be conflicting. I saw a lot of isssues here were people have reported using some of the mods I'm using but the issues they are having aren't the same as mine. I will do further testing in a few hours. 

https://www.curseforge.com/minecraft/modpacks/blepquest

**Comment by Johni0702** [comment]
Please try using BP 0.2.x, lots of stuff has changed internally since 0.1.
If that makes no differences, 1. and 2. from above still apply.

Also, FYI editing issues on GitHub does not notify other people. When you've got new stuff be sure to post a new comment.

**Comment by hellot1m** [comment]
Fixed the netty issue. When adding this mod to our server, we had lots of issues getting the server stable but found that it continued to get worse and worse. Luckily the server was fairly new, we took a vote and started a new world. Now everything is working flawlessly :) you can close this, delete this or whatever. Thank you so much for trying to help :) maybe this could still be beneficial info for you.

**Comment by Johni0702** [comment]
I'm aware of some performance issues which (although they should theoretically exist in SP as well) appear on dedicated servers: #150 
Those are fixed in-dev and will be in 0.2.4 once I find the time to release it (if you know how, you can also build the mod from source to get them early).
